integration/auction_test.sh

```diff
@@ -45,8 +45,8 @@ auction_test() {
     try ${PEER_CMD} lifecycle chaincode commit -o ${ORDERER_ADDR} -C ${CHAN_ID} --name ${CC_ID} --version ${CC_VER} --sequence ${CC_SEQ} --signature-policy ${CC_EP}
 
     # first call negated as it fails
-    try_fail ${PEER_CMD} lifecycle chaincode createenclave -o ${ORDERER_ADDR} --name wrong-cc-id
-    try ${PEER_CMD} lifecycle chaincode createenclave -o ${ORDERER_ADDR} --name ${CC_ID}
+    try_fail ${PEER_CMD} lifecycle chaincode createenclave -o ${ORDERER_ADDR} --peerAddresses "localhost:7051" --name wrong-cc-id
```

Comment:
ok, the need for this should be obsoleted by `SGX_CREDENTIALS_PATH` (at least if we modify `config.mk` as suggested above ...)

Reply:
the default is simulation mode, and you don't have to pass the sgx-creds.
The test does not fail because it's missing the sgx-creds, rather because of the the ccid

Reply:
well, it will _also_ fail because sgx-credentials-path is missing when you run in `SGX_MODE=HW`?  In any case, it wouldn't/couldn't test what you would want to in that case due to lack of credentials?

Reply:
I modified it as we discussed. It was a great suggestion for automated testing.